### PR TITLE
Redirect new apps to form settings page

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?165622

### CODE REVIEW

**Comment by orangejenny** [comment]
@nickpell

**Comment by nickpell** [comment]
waiting on tests

File: corehq/apps/app_manager/views.py [hunk 1]
```diff
@@ -1266,13 +1270,14 @@ def default_new_app(request, domain):
         domain, _("Untitled Application"), lang=lang,
         application_version=APP_V2
     )
-    app.add_module(Module.new_module(_("Untitled Module"), lang))
-    app.new_form(0, "Untitled Form", lang)
+    module = Module.new_module(_("Untitled Module"), lang)
```
Comment (benrudolph) [inline corehq/apps/app_manager/views.py]: do you know why this `new_module` call translates "Untitled Module" and none of the rest do? not blocking just curious

Comment (orangejenny) [inline corehq/apps/app_manager/views.py]: huh, I do not. Nor did I know that the underscore translates a string, so thanks for that.

Comment (czue) [inline corehq/apps/app_manager/views.py]: it's a (very prevalent) convention but not built into the language or anything: http://commcare-hq.readthedocs.org/en/latest/translations.html

**Pull request merged**